plugins/scaffolder-backend/package.json

```diff
@@ -17,9 +17,16 @@
     "supertest": "^4.0.2"
   },
   "dependencies": {
+    "@backstage/backend-common": "0.1.1-alpha.4",
+    "dockerode": "^3.2.0",
     "express": "^4.17.1",
     "fs-extra": "^9.0.0",
-    "glob": "^7.1.6",
+    "globby": "^11.0.0",
+    "compression": "^1.7.4",
```

Comment:
Would be nice if we don't need to list all different deps for 80% of the backends, and just rely on `backend-common`. Side-note, not to do with this PR really

Reply:
Yeah me and @freben should sync.